server/app/api.py

```diff
@@ -1821,7 +1763,8 @@ def get_courses(self, course, user, data):
     def get_students(self, course, user, data):
         query = models.Participant.query(
             models.Participant.course == course.key,
-            models.Participant.role == 'student')
+            models.Participant.role == 'student',
+            models.Participant.status != 'inactive')
```

Comment:
Mhm. I was guessing something along those lines. There are actually quite a few; I can go through tomorrow, and pull them out one-by-one.